client/src/components/generic/Table.js

```diff
@@ -47,16 +53,128 @@ const StyledTableRow = withStyles((theme) => ({
   },
 }))(TableRow);
 
-export const Table = ({ order, orderBy, renderCell, onRequestSort, columns, rows, isLoading, rowsPerPage = 10 }) => {
+const useStyles1 = makeStyles((theme) => ({
```

Comment:
Typically, a bad sign when things have numbered suffixes. Kinda like `console.log` lines being added to the codebase.